pgmpy/factors/CPD.py

```diff
@@ -116,15 +119,33 @@ def __init__(self, variable, variable_card, values,
 
         super(TabularCPD, self).__init__(variables, cardinality, values.flatten('C'))
 
-    def get_cpd(self):
+    def get_cpd(self, pretty=True):
         """
         Returns the cpd
         """
         if self.variable in self.variables:
+            if pretty:
+                table_list = []
+                for i in range(len(self.cardinality))[1:]:
+                    var_rep = np.prod(self.evidence_card[i:])
```

Comment:
Don't compute the product in each loop. It would be better to use ```np.cumprod``` method and save these values apriori to the loop.